UnitTests/ConsoleDrivers/KeyEventArgsTests.cs

```diff
@@ -0,0 +1,238 @@
+	{
+		var eventArgs = new KeyEventArgs ();
+		Assert.Equal (Key.Unknown, eventArgs.Key);
+	}
+
+	[Theory]
+	[InlineData (Key.Enter)]
+	[InlineData (Key.Esc)]
+	[InlineData (Key.A)]
+	public void Constructor_WithKey_ShouldSetCorrectKey (Key key)
+	{
+		var eventArgs = new KeyEventArgs (key);
+		Assert.Equal (key, eventArgs.Key);
+	}
+
+	[Fact]
+	public void HandledProperty_ShouldBeFalseByDefault ()
+	{
+		var eventArgs = new KeyEventArgs ();
+		Assert.False (eventArgs.Handled);
+	}
+
+	[Theory]
+	[InlineData (Key.A, true)]
+	[InlineData (Key.A | Key.ShiftMask, true)]
+	[InlineData (Key.F, true)]
+	[InlineData (Key.F | Key.ShiftMask, true)]
+	[InlineData (Key.A | Key.CtrlMask, false)]
+	[InlineData (Key.A | Key.AltMask, false)]
+	[InlineData (Key.D0, false)]
+	[InlineData (Key.Esc, false)]
+	[InlineData (Key.Tab, false)]
+	public void IsAlpha (Key key, bool expected)
+	{
+		var eventArgs = new KeyEventArgs (key);
+		Assert.Equal (expected, eventArgs.IsAlpha);
+	}
+
+	[Theory]
+	[InlineData ((Key)'❿', '❿')]
+	[InlineData ((Key)'☑', '☑')]
+	[InlineData ((Key)'英', '英')]
+	[InlineData ((Key)'{', '{')]
+	[InlineData ((Key)'\'', '\'')]
+	[InlineData ((Key)'\r', '\r')]
+	[InlineData ((Key)'ó', 'ó')]
+	[InlineData ((Key)'ó' | Key.ShiftMask, 'ó')]
+	[InlineData ((Key)'Ó', 'Ó')]
+	[InlineData ((Key)'ç' | Key.ShiftMask | Key.AltMask | Key.CtrlMask, default)]
+
+	[InlineData ((Key)'a', 97)] // 97 or Key.Space | Key.A
+	[InlineData ((Key)'A', 97)] // 65 or equivalent to Key.A, but A-Z are mapped to lower case by drivers
+	//[InlineData (Key.A, 97)] // 65 equivalent to (Key)'A', but A-Z are mapped to lower case by drivers
+	[InlineData (Key.ShiftMask | Key.A, 65)]
+	[InlineData (Key.CtrlMask | Key.A, default)]
+	[InlineData (Key.AltMask | Key.A, default)]
+	[InlineData (Key.ShiftMask | Key.CtrlMask | Key.A, default)]
+	[InlineData (Key.ShiftMask | Key.AltMask | Key.A, default)]
+	[InlineData (Key.AltMask | Key.CtrlMask | Key.A, default)]
+	[InlineData (Key.ShiftMask | Key.CtrlMask | Key.AltMask | Key.A, default)]
+	
+	[InlineData ((Key)'z', 'z')]
+	[InlineData ((Key)'Z', 'z')]
+	[InlineData (Key.ShiftMask | Key.Z, 'Z')]
+
+	[InlineData ((Key)'1', '1')]
+	[InlineData (Key.ShiftMask | Key.D1, '1')]
+	[InlineData (Key.CtrlMask | Key.D1, default)]
+	[InlineData (Key.ShiftMask | Key.CtrlMask | Key.D1, default)]
+
+	[InlineData (Key.F1, default)]
+	[InlineData (Key.ShiftMask | Key.F1, default)]
+	[InlineData (Key.CtrlMask | Key.F1, default)]
+
+	[InlineData (Key.Enter, '\n')]
+	[InlineData (Key.Tab, '\t')]
+	[InlineData (Key.Esc, 0x1b)]
+	[InlineData (Key.Space, ' ')]
+
+	[InlineData (Key.ShiftMask | Key.CtrlMask | Key.AltMask | Key.Enter, default)]
+
+	[InlineData (Key.Null, default)]
+	[InlineData (Key.ShiftMask | Key.CtrlMask | Key.AltMask | Key.Null, default)]
+
+	[InlineData (Key.CharMask, default)]
+	[InlineData (Key.SpecialMask, default)]
+	public void AsRune_ShouldReturnCorrectIntValue (Key key, Rune expected)
```

Comment:
To avoid this warning change all the `default` to `'\0'`.
`Null should not be used for type parameter 'expected' of type 'System.Text.Rune'. Use a non-null value, or convert the parameter to a nullable type.`

```cs
[InlineData (Key.SpecialMask, '\0')]
```